packages-web/maps/webpack.config.dev.js

```diff
@@ -1,15 +1,22 @@
 const merge = require("webpack-merge");
-const baseConfig = require("../utils-react-widgets/configs/webpack.config.dev");
+const baseConfig = require("../../configs/webpack.dev.js");
 
 const customConfig = {
+    output: {
+        filename: "widgets/com/mendix/widget/custom/Maps/Maps.js"
+    }
+};
+
+const customPreviewConfig = {
     module: {
-        rules: [
-            {
-                test: /\.(png|jpg|gif|svg|eot|ttf|woff|woff2)$/,
-                loader: "url-loader"
-            }
-        ]
+        rules: [{ test: /\.(png|jpg|gif|svg|eot|ttf|woff|woff2)$/, loader: "url-loader?limit=100000" }]
```

Comment:
Do we have images in this package?

Reply:
Yes, leaflet exports some images (inside css) and we need to use it for preview and also base64 for prod